README.md

```diff
@@ -10,9 +10,33 @@
 Small package to visualize your [Ferrite.jl](https://github.com/Ferrite-FEM/Ferrite.jl) results. Currently supports only Makie,
 but the goal is to extend it for different plotting packages.
 
+The package is highly experimental and breaking changes about the internal machinery are about to come.
+Likely, only a small fraction of the interface will change over time.
+
 ## Installation
 
 ```julia
 pkg> add FerriteVis
 ```
 
+
+## Features
+
+- `solutionplot` FE solution contour plot on arbitrary finite element mesh (in Makie called `mesh` plots)
+- `ferriteviewer` viewer with toggles and menus that update the plot
+- `wireframe` plots the finite element mesh and optionally labels nodes and cells
+- `arrows` - also called `quiver` plots, in paraview `glyph` filter
+- `surface` 2D solutions in 3D space as surface, in paraview `warp by scalar` filter
+- live plotting while your simulation runs with any of the above listed options
+- mutating versions of the above listed functions (except for the viewer)
+- deformed plots available for `solutionplot` and `wireframe` deformed plots available for `solutionplot` and `wireframe` 
+- full integration into the Makie ecosystem, e.g. themes, layouts etc. 
+- GPU powered plotting with GLMakie.jl, jupyter/pluto notebook plotting with WGLMakie.jl and vector graphics with CairoMakie.jl
+
+## Missing Features
+
+- correct visualization of nonlinear geometry faces/edges
+- visualization of boundary conditions
+- subdomain entity plotting, e.g. cellsets, facesets, edgesets and so on
```

Comment:
Probably we should also refer to the missing MixedDofHandler integration here.

Reply:
`MixedDofHandler` in the same way `DofHandler` does for fields that are defined on the whole domain, so I think stating that subdomain entity plotting is missing is correct